.github/workflows/typespec-latest-validation.yml

```diff
@@ -0,0 +1,91 @@
+name: TypeSpec Latest Validation
+
+on:
+  issue_comment:
+    types: [created]
+  workflow_dispatch:
+    inputs:
+      typespec_ref:
+        description: 'TypeSpec branch/ref to test against'
+        required: false
+        default: 'main'
+        type: string
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  typespec-latest-validation:
+    name: Validate against TypeSpec Latest
+    runs-on: ubuntu-latest
+    if: |
+      (github.event_name == 'workflow_dispatch') ||
+      (github.event_name == 'issue_comment' && 
+       github.event.issue.pull_request &&
+       contains(github.event.comment.body, '/integration azure-specs') &&
+       (github.event.comment.author_association == 'MEMBER' || 
+        github.event.comment.author_association == 'OWNER' || 
+        github.event.comment.author_association == 'COLLABORATOR'))
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+          fetch-depth: 0
+          token: ${{ github.token }}
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+
+      - name: Checkout Azure REST API specs
+        uses: actions/checkout@v4
+        with:
+          repository: Azure/azure-rest-api-specs
+          path: azure-rest-api-specs
+          fetch-depth: 1
+
+      - name: Setup Node.js and pnpm
+        uses: ./.github/actions/setup
+        with:
+          node-version: "20.x"
+
+      - name: Update core submodule to latest TypeSpec
+        run: |
+          cd core
+          git fetch origin
+          TYPESPEC_REF="${{ github.event.inputs.typespec_ref || 'main' }}"
+          echo "Testing against TypeSpec ref: $TYPESPEC_REF"
+          git checkout origin/$TYPESPEC_REF
+          cd ..
+          git add core
+
+      - name: Install dependencies
+        run: pnpm install --no-frozen-lockfile
+
+      - name: Build core TypeSpec packages
+        run: |
+          cd core
+          pnpm build:all
+
+      - name: Build TypeSpec Azure packages
+        run: pnpm build
+        continue-on-error: true
+
+      - name: Generate compatibility report
```

Comment:
This is missing the whole point of this workflow which is to run typespec validation in the spec repo